tests/e2e/modify_volume.go

```diff
@@ -123,9 +144,11 @@ var _ = Describe("[ebs-csi-e2e] [single-az] [modify-volume] Modifying a PVC", fu
 	for testName, modifyVolumeTest := range modifyVolumeTests {
 		modifyVolumeTest := modifyVolumeTest
 		Context(testName, func() {
-			It("will modify associated PV and EBS Volume via volume-modifier-for-k8s", func() {
-				modifyVolumeTest.Run(cs, ns, ebsDriver, testsuites.VolumeModifierForK8s)
-			})
+			if !modifyVolumeTest.ExternalResizerOnly {
```

Comment:
Skipping implies there are some circumstances under which the test would run - because this test will never run, I would strongly prefer the way it's currently done.

(Skipping also would involve plumbing in the test config or some weirdness like that)

Reply:
If you disagree, then I'm fine with keeping it as is.

But to be fair the subtests themselves will already only run in certain circumstances. When ExternalResizerOnly is false. I believe other projects prefer using Skip() within deepest subtest over this current style for better readability.

And we already have `ExternalResizerOnly` plumbing don't we?

Reply:
Ah yes, we do, I misunderstood how we were passing the test. In that case I don't care that much, I'll leave the decision up to Rafael.

Reply:
Decided to use the Skip() function, gives better readability